### ISSUE TITLE
in chat tab,Need a "Create A New Chat" button on the Chat page

### ISSUE THREAD

**Issue by zrwd01** [bitswired/ai-chat-app#4]
Hey bro, you have worked hard

in chat tab,Need a "Create A New Chat" button on the Chat page

and when "Start Bisiting The Chat Page to Create A New Chat" button on the first page is pressed, and enter a wrong 404 page.

**Comment by jimzer** [comment]
Hey @zrwd01! 
Thank you for raising the issue.
I hope you like the app :)

I pushed a new major version with lots of improvements.
I added the button to create the chat on the Chat page and I fixed the issue in the home page.

Be careful, you now have to add your API key in the /settings page.

The new features are:

👉🏽 Import & create templates
👉🏽 Create Powerful Smart templates
👉🏽 Dark Mode
👉🏽 Render Markdown
👉🏽 Refreshed UI Design

I updated the video in the README to introduce the new changes.
